src/Dapr.Actors/Runtime/ActorRuntime.cs

```diff
@@ -139,6 +143,16 @@ internal Task FireTimerAsync(string actorTypeName, string actorId, string timerN
             }
         }
 
+        internal static IDaprInteractor GetDaprInteractor(string actorTypeName)
```

Comment:
What does this change accomplish that's better than the property we had here before? I don't see why we need a separate interactor per-actor-type.